### PR TITLE
[model_cards] Add a new model for Irish

### PR DESCRIPTION
**Pull request by jimregan**

### CODE REVIEW

File: model_cards/jimregan/BERTreach/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+## BERTreach
+
+([beirtreach](https://www.teanglann.ie/en/fgb/beirtreach) means 'oyster bed')
+
+**Model size:** 84M
+
+**Training data:** 
+* [PARSEME 1.2](https://gitlab.com/parseme/parseme_corpus_ga/-/blob/master/README.md) 
+* Newscrawl 300k portion of the [Leipzig Corpora](https://wortschatz.uni-leipzig.de/en/download/irish)
+* Private news corpus crawled with [Corpus Crawler](https://github.com/google/corpuscrawler)
```
Comment (julien-c) [inline model_cards/jimregan/BERTreach/README.md:16]: How large was the total dataset?

File: model_cards/jimregan/BERTreach/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+language: ga
```
Comment (julien-c) [inline model_cards/jimregan/BERTreach/README.md:2]: Did you get a chance to test the Opus-MT translation models for Irish? https://huggingface.co/models?filter=ga

Are they good?

**Comment by julien-c** [comment]
Really cool! If you get a chance, do you think you could add sample inputs for Irish to https://github.com/huggingface/widgets-server/blob/master/DefaultWidget.ts?

File: model_cards/jimregan/BERTreach/README.md (hunk 1 above)
Comment (jimregan) [reply model_cards/jimregan/BERTreach/README.md:16]: 2125804  sentences, 47419062 (wc) tokens. Pretty small, but a lot bigger than the one from Turku.

Comment (julien-c) [reply model_cards/jimregan/BERTreach/README.md:16]: Feel free to add to the model card! (we'll make it way easier to update in the next few weeks)

File: model_cards/jimregan/BERTreach/README.md (hunk 2 above)
Comment (jimregan) [reply model_cards/jimregan/BERTreach/README.md:2]: I haven't looked. I worked with MT for several years, did a master's on it and burned myself out completely writing the dissertation, so I wasn't really inclined to check. 
I get 'Can't load config' on both models from the website; the English->Irish model seems good enough for the sort of short sentences I can confidently rate.

Comment (julien-c) [reply model_cards/jimregan/BERTreach/README.md:2]: Understood, thanks!

cc @sshleifer and @mfuntowicz for the inference API issue

**Pull request merged**